onebusaway-android/build.gradle

```diff
@@ -272,7 +275,6 @@ tasks.whenTaskAdded { theTask ->
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: '*.jar')
```

Comment:
This line is actually unrelated to the Amazon library - it includes any bundled JAR files in `onebusaway-android/libs`, which doesn't currently exist.  I'm fine with removing it, though, as its currently unused.